src/lib/mongoBackend/mongoNotifyContext.cpp

```diff
@@ -43,7 +43,9 @@ HttpStatusCode mongoNotifyContext
   const std::vector<std::string>&  servicePathV
 )
 {
-    reqSemTake(__FUNCTION__, "ngsi10 notification");
+    bool reqSemTaken;
+
+    reqSemTake(__FUNCTION__, "ngsi10 notification", SemReadOp, &reqSemTaken);
```

Comment:
I think mongoNotifyContext refers to the processing of notifyContextRequests. Thus, it is like a updateContext. In that case, SemWriteOp should be used.